develop/develop/settings.py

```diff
@@ -160,8 +161,8 @@
 
 # Vendor Promo Settings
 VENDOR_PROMO_PROCESSOR = os.getenv("VENDOR_PROMO_PROCESSOR", "base.PromoProcessorBase")
-VENDOR_PROMO_PROCESSOR_URL = os.getenv("VENDOR_PROMO_PROCESSOR_URL")
-VENDOR_PROMO_PROCESSOR_BARRER_KEY = os.getenv("VENDOR_PROMO_PROCESSOR_BARRER_KEY")
+VENDOR_PROMO_PROCESSOR_URL = os.getenv("VENDOR_PROMO_PROCESSOR_URL", None)
+VENDOR_PROMO_PROCESSOR_BARRER_KEY = os.getenv("VENDOR_PROMO_PROCESSOR_BARRER_KEY", None)
```

Comment:
definitely not a blocker, but os.getenv already returns None as its default

Reply:
@mgcamisa this is known, but I prefer this explicit style of default definition, as its more visually useful in `if env_var_defined:` style use cases, and more resilient against upstream library defaults changes over time.

Reply:
Yes, that is exactly why I did it.